setup.cfg

```
@@ -20,13 +20,13 @@ classifiers =
 [options]
 python_requires = >=3.5
 install_requires =
-    niworkflows @ git+https://github.com/poldracklab/niworkflows.git@6118af2d8ef59abfa79f665da7caad646c40bc5c
+    niworkflows @ git+https://github.com/tsalo/niworkflows.git@14d33d560643f462dc80d4de8fbcca15a7713d25
     smriprep ~= 0.3.2
     templateflow ~= 0.4.1
     nibabel >=2.2.1
     indexed_gzip >=0.8.8
     nilearn !=0.5.0, !=0.5.1
-    nipype >=1.2.2
+    nipype @ git+https://github.com/effigies/nipype.git@e206f3d53198c683e959fe1e73426ce5cf362b93
```

Sorry, minor fix upstream.

```suggestion
    nipype @ git+https://github.com/effigies/nipype.git@26b662f0ff48886524577c83da22a0bc68189a08
```